UM/Qt/qml/UM/CheckBox.qml

```diff
@@ -46,20 +51,37 @@ CheckBox
                 State {
                     name: "unchecked"
                     when: control.enabled && control.checkState == Qt.Unchecked
-                    PropertyChanges { target: indicator_item; width: Math.round(parent.width / 1.5); height: width }
-                    PropertyChanges { target: indicator_item; color: UM.Theme.getColor("checkbox_mark"); source: UM.Theme.getIcon("Check", "low"); opacity: false; }
+                    PropertyChanges
+                    {
+                        target: indicator_item
+                        width: Math.round(parent.width / 1.5)
+                        height: width
+                        opacity: false
```

Comment:
this was there before but this might be a good time to tackle this as well.

- why set width and height if the property is not visible
- its a bit weird to set opacity to a bool as the property is a real https://doc.qt.io/qt-5/qml-qtquick-item.html#opacity-prop. I have two suggetsions

```suggestion
                        visible: false
```

```suggestion
                        source: ""
```